packages/pn-personafisica-webapp/src/component/Contacts/__test__/CourtesyContactItem.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import MockAdapter from 'axios-mock-adapter';
```

Comment:
if you define let mock, in each you must delete const mock and leave only mock.
The afterEach method runs code efter each test, so the mock.restore and mock.reset must be deleted from the single test. 
Please check all tests